src/TaskList.js

```diff
@@ -0,0 +1,15 @@
+import Task from './Task.js';
+
+
+const TaskList = (props) => {
+    const data = props.showAllTasks ? props.data : props.data.filter(entry => !entry.completed);
```

Comment:
If you did this in ToggleBar, there'd be no need to pass down props.showAllTasks